src/components/_modal/pages/login/Select.js

```diff
@@ -0,0 +1,73 @@
+import animations from "css/animations";
```

Comment:
en lille kommentar til filen a'la 
@file Select.js  - show a branch to be selected